docs/iris/gallery_code/oceanography/plot_load_nemo.py

```diff
@@ -7,12 +7,13 @@
 different time dimensions in these files can prevent Iris from concatenating
 them without the intervention shown here.
 """
+
 from __future__ import unicode_literals
+import matplotlib.pyplot as plt
```

Comment:
@tkknight Can you inject a blank link here between line `11` and line `12`, thanks